content/docs/command-reference/update.md

```diff
@@ -1,7 +1,7 @@
 # update
 
-Update files or directories imported from external <abbr>DVC
-repositories</abbr>, and the corresponding import `.dvc` files.
+Update files or directories imported from external <abbr>DVC repositories</abbr>
+or URLs, and the corresponding import `.dvc` files.
```

Comment:
Maybe we should have a `<abbr>` also for URLs or link to the type of URL supported in this case (https://dvc.org/doc/command-reference/import-url#description)